packages/main/src/Table.js

```diff
@@ -49,6 +49,28 @@ const metadata = {
 		},
 	},
 	properties: /** @lends sap.ui.webcomponents.main.Table.prototype */ {
+
+		/**
+		 * Defines the text that will be displayed when there is no data and <code>show-no-date</code> is present.
+		 *
+		 * @type {String}
+		 * @public
+		 */
+		noDataText: {
+			type: String,
+		},
+
+		/**
+		 * Defines if the value of <code>no-data-text</code> will be diplayed when there is no rows present in the table.
+		 *
+		 * @type {boolean}
+		 * @defaultvalue false
+		 * @public
+		 */
+		showNoData: {
+			type: Boolean,
+			defaultValue: false,
```

Comment:
remove ```defaultValue: false``` from the metadata, as the the default value fo booleans is always false in the framework.